packages/replay-next/src/suspense/PointStackCache.ts

```diff
@@ -0,0 +1,37 @@
+import { ExecutionPoint, PointStackFrame } from "@replayio/protocol";
+import { IntervalCache, createIntervalCache } from "suspense";
+
+import { ReplayClientInterface } from "shared/client/types";
+
+import { updateMappedLocation } from "./PauseCache";
+import { sourcesByIdCache } from "./SourcesCache";
+
+export const pointStackCache: IntervalCache<
```

Comment:
The choice to use an interval cache for this was surprising to me at first.

I guess we're always fetching frames 0...N, so the interval cache prevents us from re-fetching the lower frames if we need to step in and fetch the next one(s)?